Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -39,8 +39,8 @@ export function initialize(element, elementId, options) {
     const defaultOptions = {
         enableTime: options.inputMode === 1,
         dateFormat: options.inputMode === 1 ? 'Y-m-d H:i' : 'Y-m-d',
-        allowInput: true,
-        altInput: true,
+        allowInput: !(options.disableMobile || false),
+        altInput: !(options.disableMobile || false),
```

Comment:
Is this some kind of way to coalesce is that it? I don't get the usage of the or logic here.